README.md

```diff
@@ -98,6 +98,12 @@ docker run -d -p 80:9080 -p 443:9443 -e WLP_LOGGING_CONSOLE_FORMAT=JSON -e WLP_L
 
 For more information regarding the configuration of WebSphere Liberty's logging capabilities see: https://www.ibm.com/support/knowledgecenter/en/SSD28V_liberty/com.ibm.websphere.wlp.core.doc/ae/rwlp_logging.html
 
+
+### Security
+
+Single Sign On can be optionally configured by adding Liberty server variables in an xml file, by passing environment variables (less secure),
```

Comment:
SSO is not here yet, so maybe you can just keep a general sentence here like "Click [here] for security related configuration"

Reply:
Updated